README.md

```diff
@@ -66,7 +66,7 @@ repo-token | `secrets.GITHUB_TOKEN` | Access token for which this action will ru
 ignore-authors | dependabot<br/>dependabot[bot] | If the action detects that the PR author is one of these logins, it will skip checks and set all outputs to `true`.
 base-comment | (see [action.yml](./action.yml)) | Preamble to any comment the action leaves on the PR.
 ignore-team-members | true | Whether to ignore in the case that the PR author is a) the owner of repository, or b) has a public membership[^1] in the org that owns the repository.
-body-regex | `(fix(es\|ed)?\|((resolve\|close)(s\|d)?)) #\d*[1-9]\d*` | Regular expression to identify whether the PR body refers to an issue[^2][^3].
+body-regex | `(fix(es|ed)?|((resolve|close)(s|d)?)) (https:\/\/github\.com\/[a-zA-Z0-9\-_]+\/[a-zA-Z0-9\-_]+\/issues\/\d+)?|(#\d*[1-9]\d*?)` | Regular expression to identify whether the PR body refers to an issue[^2][^3].
```

Comment:
with the value being so long, I'd suggest having the cell content refer to action.yml as you see some of the others.

Reply:
Done!